src/pybind/mgr/dashboard/frontend/src/app/shared/pipes/search-highlight.pipe.ts

```diff
@@ -0,0 +1,26 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'searchHighlight'
+})
+export class SearchHighlightPipe implements PipeTransform {
+  transform(value: string, args: string): string {
+    if (!args) {
+      return value;
+    }
+    args = this.escapeRegExp(args);
+    const regex = new RegExp(args, 'gi');
+    const match = value.match(regex);
+
+    if (!match) {
+      return value;
+    }
+
+    return value.replace(regex, '<mark>$&</mark>');
```

Comment:
If we don't really support regex (do we?), why not simply using a [`string.replace()`](https://stackoverflow.com/questions/288368/javascript-non-regex-replace)?

Reply:
Hmm true, we don't need regex here. A simple `string.replace()` will get the job done. Will update the code.